main.pm

```diff
@@ -385,6 +389,9 @@ sub load_inst_tests() {
         else {
             loadtest "installation/skip_registration.pm";
         }
+        if (is_sles4sap) {
```

Comment:
but the "full block" would consist of only a single line and http://www.extremeprogramming.org/rules/early.html